lib/gameplay.rb

```diff
@@ -1,28 +1,66 @@
-require './lib/board'
-require './lib/cell'
-require './lib/ship'
+
 require "pry"
+
 class Gameplay
-attr_reader :board, :cruiser
-  def initialize(board)
-    @board = board
-    @cruiser = Ship.new("Cruiser", 3)
-    @submarine = Ship.new("Submarine", 2)
+attr_reader :board1, :cruiser
```

Comment:
The hit list is a good idea since it allows you to prevent the player from hitting the same spot twice